vignettes/data_protocol.Rmd

```diff
@@ -96,29 +101,29 @@ The distributions currently included in the literature search for each pathogen/
 
     Again, any papers excluded at this stage should be recorded in the database of unsuitable sources with reasoning to prevent having to reassess when updating the database.
 
-1. **Post hoc removal**: if any `epiparameter` parameters are later identified as being inappropriate then they can be removed from the database. In most cases this is unlikely as limitations can be appended onto data entries to make users aware of limitations (e.g. around assumptions used to infer the distirbution), only in the most extreme cases will data be completely removed from the database.
+1. **Post hoc removal**: if any `{epiparameter}` parameters are later identified as being inappropriate then they can be removed from the database. In most cases this is unlikely as limitations can be appended onto data entries to make users aware of limitations (e.g. around assumptions used to infer the distirbution), only in the most extreme cases will data be completely removed from the database.
 
 Note: systematic reviews focusing on effect sizes can be subject to publication bias (e.g. more positive or significant results in the literature). However, distribution inference does not focus on significance testing or effect sizes, so this bias is not considered in the collection process.
```

Comment:
This I wouldn't include if it was for contributors

Reply:
No action needed.